packages/web/src/Chip/Chip.tsx

```diff
@@ -3,12 +3,19 @@ import React, { FunctionComponent } from 'react';
 import MaterialChip from '@material-ui/core/Chip'
 
 export interface IProps {
+  avatar?: any;
+  classes?: object;
+  clickable?: boolean;
+  component?: any;
+  deleteIcon?: any;
+  disabled?: boolean;
+  icon?: any;
   label: string;
-  onDelete: any;
+  onDelete?: void;
```

Comment:
The right syntax is 
`onDelete?: () => void`

Reply:
updated